ml-agents/mlagents/trainers/torch/encoders.py

```diff
@@ -180,10 +182,12 @@ def __init__(self, height, width, initial_channels, output_size):
         self.dense = nn.Linear(self.final_flat, self.h_size)
 
     def forward(self, visual_obs):
-        conv_1 = torch.relu(self.conv1(visual_obs))
-        conv_2 = torch.relu(self.conv2(conv_1))
-        conv_3 = torch.relu(self.conv3(conv_2))
-        hidden = torch.relu(self.dense(conv_3.view([-1, self.final_flat])))
+        conv_1 = nn.functional.leaky_relu(self.conv1(visual_obs))
```

Comment:
Arn't we using swish in the TensorFlow code ?

Reply:
Yes - this is just to get the tests to pass (lots of zero gradients if you use ReLU). Will do a separate PR to change everything to Swish (requires some refactor of `utils.py` to avoid a circular import in the `encoders.py`.)